### PR TITLE
Feature/update UI

### PR DESCRIPTION
**Pull request by madhaus4**
# Description
- Add delete functionality for film fam
- Style film fam to layout version 1
- Import logo to start styling login page
- Adjust edit button locations
- Render images instead of forms on Edit page load

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?
- [x] Use console.log()
- [x] Dev tools / react dev tools
- [x] View in local host
- [ ] Verify acceptance criteria (if applicable)

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by rachelJensen — APPROVED** [review]
I think this looks great. I'm going to create another issue, however, to remind us to do a clean-up of commented out code which there is a still a lot of in this code base.

File: src/components/Filmteam/FilmTeam.scss [hunk 1]
```diff
@@ -2,36 +2,47 @@
 @import '../../styles/mixins';
 
 .film-team-container {
-  height: 45vh;
-  width: 100%;
-  padding: 2vh;
+  grid-column-start: 1;
+  grid-column-end: 5;
+  background-color: whitesmoke;
```
Comment (ninabrissey) [inline src/components/Filmteam/FilmTeam.scss:7]: As we think about the layout of the page, we may end up needing to put this `whitesmoke` on opposite components.

File: src/components/Filmteam/FilmTeam.scss [hunk 2]
```diff
@@ -60,23 +71,52 @@
 .all-crew-wrapper {
   @include gridDisplayCards;
   @include marginWidthContainers;
+  justify-items: center;
 }
 
 
 .crew-member-card {
+  // height: 17vh;
```
Comment (ninabrissey) [inline src/components/Filmteam/FilmTeam.scss:79]: Shall we remove the height and width.

**Comment by ninabrissey** [comment]
We should look at the organization of the components because re-ordering them is 99% likely.

**Pull request merged**